src/demo/demo.js

```diff
@@ -30,7 +30,7 @@ let url = null
 const defaultRegion = 'EU'
 
 if (process.env.NODE_ENV === 'development') {
-  require('preact/devtools')
+  require('preact/debug')
```

Comment:
@stefaniacardenas we need to put this `require` on top of `src/index.js` to enable full DOM inspection:
![2020-10-21_14-20-50](https://user-images.githubusercontent.com/1215541/96726725-2b7e5600-13aa-11eb-8fb0-064a66061356.png)

Reply:
Thank you @ethan605 🙏